patches/extensions-common-permissions-api_permission.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/extensions/common/permissions/api_permission.h b/extensions/common/permissions/api_permission.h
+index 6354d5c66b254779a9a1a10816a9cf35a1df99b8..8630a5e5dca68c5cd6803136e5163c7edfbc09e1 100644
+--- a/extensions/common/permissions/api_permission.h
++++ b/extensions/common/permissions/api_permission.h
+@@ -272,6 +272,7 @@ class APIPermission {
+     kSearch = 228,
+     kTabGroups = 229,
+     kScripting = 230,
++    BRAVE_ID
```

Comment:
Do these need to be consecutive or can we leave a gap so that we don't collide with future Chromium additions?

Reply:
added empty line

Reply:
Sorry, I meant the actual numeric value (231). But it looks like the values are used in a loop in `ExtensionService::RecordPermissionMessagesHistogram`, so a gap in numbers wouldn't work.

We actually do not want to add empty lines in our patches.